### ISSUE TITLE
[Paginator] Low Contrast Ratio for "Current Page" Indicator

### ISSUE THREAD

**Issue by DanielleRameau** [sl-design-system/components#2708]
### Provide a general summary of the issue here

**Description:**
The "current page" indicator (blue square) in the paginator has a contrast ratio of only 1.3:1 against the background. This is below the minimum recommended contrast ratio of 3:1, making it difficult to visually identify the active page.

**Expected Behavior:**
The current page indicator should have a contrast ratio of at least 3:1 to meet accessibility standards.

**Tasks**

- [ ] Improve design in figma component
- [ ] implement in code

### 👤 Your name

DR

### 🧢 Your Product/Team

SLDS

**Comment by DanielleRameau** [comment]
Selected state doesn't work in Storybook right now, could you have a look at that as well?
